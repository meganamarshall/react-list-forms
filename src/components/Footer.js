import React from 'react';
import styles from './Header.css';

export default function Footer() {
  return (
    <footer className={styles.Footer}>
      <p>By Megan 2019</p>
    </footer>
  );
}
