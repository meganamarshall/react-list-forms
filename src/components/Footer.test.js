import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('footer component', () => {
  it('renders the footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
