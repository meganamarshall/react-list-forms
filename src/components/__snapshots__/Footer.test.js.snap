// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer component renders the footer 1`] = `
<footer
  className="Header"
>
  <p>
    By Megan 2019
  </p>
</footer>
`;
